dash/dash-renderer/src/actions/callbacks.ts

```diff
@@ -334,6 +334,11 @@ function updateComponent(component_id: any, props: any) {
     return function (dispatch: any, getState: any) {
         const paths = getState().paths;
         const componentPath = getPath(paths, component_id);
+        if (typeof componentPath === 'undefined') {
+            // Can't find the component that was defined in the running keyword,
+            // Let's skip the component to prevent the dashboard from crashing.
```

Comment:
I think it would be better to throw an error unless `config.suppress_callback_exceptions=True`. It is in state.config, could refactor line 335 to `const {paths,config} = getState();`

Reply:
I added a check for that and threw an error when suppress_callback_exceptions=False